common.h

```diff
@@ -0,0 +1,30 @@
+#include<stdio.h>
+#include<stdint.h>
+//#include<cheri/cheri.h>
```

Comment:
Generally we don't want to include commented code.

Reply:
Fixed in: https://github.com/capablevms/cheri-examples/pull/1/commits/1f8537c5e8e99633fb816022662eb0d728c14d84